main.go

```diff
@@ -26,13 +26,15 @@ var (
 )
 
 type HelmController struct {
-	Kubeconfig     string `short:"k" usage:"Kubernetes config files, e.g. $HOME/.kube/config" env:"KUBECONFIG"`
-	MasterURL      string `short:"m" usage:"Kubernetes cluster master URL" env:"MASTERURL"`
-	Namespace      string `short:"n" usage:"Namespace to watch, empty means it will watch CRDs in all namespaces." env:"NAMESPACE"`
-	Threads        int    `short:"t" usage:"Threadiness level to set, defaults to 2." default:"2" env:"THREADS"`
-	ControllerName string `usage:"Unique name to identify this controller that is added to all HelmCharts tracked by this controller" default:"helm-controller" env:"CONTROLLER_NAME"`
-	NodeName       string `usage:"Name of the node this controller is running on" env:"NODE_NAME"`
-	PprofPort      int    `usage:"Port to publish HTTP server runtime profiling data in the format expected by the pprof visualization tool. Only enabled if in debug mode" default:"6060"`
+	Kubeconfig     string `short:"k" usage:"Kubernetes config files, e.g. $HOME/.kube/config. May set via KUBECONFIG env var." env:"KUBECONFIG"`
+	MasterURL      string `short:"m" usage:"Kubernetes cluster master URL. May set via MASTERURL env var." env:"MASTERURL"`
+	Namespace      string `short:"n" usage:"Namespace to watch, empty means it will watch CRDs in all namespaces. May set via NAMESPACE env var." env:"NAMESPACE"`
+	Threads        int    `short:"t" usage:"Threadiness level to set. May set via THREADS env var." default:"2" env:"THREADS"`
+	ControllerName string `usage:"Unique name to identify this controller that is added to all HelmCharts tracked by this controller. May set via CONTROLLER_NAME env var." default:"helm-controller" env:"CONTROLLER_NAME"`
+	NodeName       string `usage:"Name of the node this controller is running on. May set via NODE_NAME env var." env:"NODE_NAME"`
+	JobClusterRole string `usage:"Name of the cluster role to use by jobs managing helm charts. Perms must be the same or less compared to helm controller. May set via JOB_CLUSTER_ROLE env var." default:"cluster-admin" env:"JOB_CLUSTER_ROLE"`
+	JobImage       string `usage:"Image to use by jobs managing helm charts. May set via JOB_IMAGE env var." default:"rancher/klipper-helm:v0.8.2-build20230815" env:"JOB_IMAGE"`
```

Comment:
I'm not sure what `Perms must be the same or less compared to helm controller` is supposed to mean - can you clarify this, or perhaps omit it from here entirely and we can work on documenting the various tunable parameters?

Also, nit on the grammar of the help text for env vars.

```suggestion
	Kubeconfig     string `short:"k" usage:"Kubernetes config files, e.g. $HOME/.kube/config. May be set via KUBECONFIG env var." env:"KUBECONFIG"`
	MasterURL      string `short:"m" usage:"Kubernetes cluster master URL. May be set via MASTERURL env var." env:"MASTERURL"`
	Namespace      string `short:"n" usage:"Namespace to watch, empty means it will watch CRDs in all namespaces. May set via NAMESPACE env var." env:"NAMESPACE"`
	Threads        int    `short:"t" usage:"Threadiness level to set. May be set via THREADS env var." default:"2" env:"THREADS"`
	ControllerName string `usage:"Unique name to identify this controller that is added to all HelmCharts tracked by this controller. May be set via CONTROLLER_NAME env var." default:"helm-controller" env:"CONTROLLER_NAME"`
	NodeName       string `usage:"Name of the node this controller is running on. May be set via NODE_NAME env var." env:"NODE_NAME"`
	JobClusterRole string `usage:"Name of the cluster role to use for jobs created to manage helm charts. May be set via JOB_CLUSTER_ROLE env var." default:"cluster-admin" env:"JOB_CLUSTER_ROLE"`
	JobImage       string `usage:"Image to use by jobs managing helm charts. May be set via JOB_IMAGE env var." default:"rancher/klipper-helm:v0.8.2-build20230815" env:"JOB_IMAGE"`
```

Reply:
Just so you know, the issue with permissions is that Helm Controller cannot run Jobs with permissions higher than it has. Kubernetes returns me an error like the following:

> level=error msg="error syncing 'hec-stubs/podinfo': handler helm-controller-chart-registration: failed to create helm-hec-stubs-podinfo rbac.authorization.k8s.io/v1, Kind=ClusterRoleBinding for helm-controller-chart-registration hec-stubs/podinfo: clusterrolebindings.rbac.authorization.k8s.io \"helm-hec-stubs-podinfo\" is forbidden: user \"system:serviceaccount:hec-local:hec-helm-controller\" (groups=[\"system:serviceaccounts\" \"system:serviceaccounts:hec-local\" \"system:authenticated\"]) is attempting to grant RBAC permissions not currently held:\n{APIGroups:[\"*\"], Resources:[\"*\"], Verbs:[\"*\"]}\n{NonResourceURLs:[\"*\"], Verbs:[\"*\"]}, requeuing"

Thanks for the rest of the comments. I will take them into account or refer to them a little later. Time to relax :)

Reply:
Yeah, it is not allowed to escalate privileges that way - but the phrasing is a little weird, and I'm not sure we need to pack all the warnings into the help test.